internal/services/codesigning/trusted_signing_certificate_profile_resource_test.go

```diff
@@ -0,0 +1,245 @@
+				check.That(data.ResourceName).Key("certificates.0.status").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.subject_name").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.thumbprint").Exists(),
+				check.That(data.ResourceName).Key("status").Exists(),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccTrustedSigningCertificateProfile_requiresImport(t *testing.T) {
+	if os.Getenv("ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID") == "" {
+		t.Skip("Skipping as ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID is not specified")
+		return
+	}
+	data := acceptance.BuildTestData(t, "azurerm_trusted_signing_certificate_profile", "test")
+	r := TrustedSigningCertificateProfileResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("certificates.0.created_date").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.enhanced_key_usage").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.expiry_date").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.serial_number").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.status").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.subject_name").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.thumbprint").Exists(),
+				check.That(data.ResourceName).Key("status").Exists(),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func TestAcccodesigningCertificateProfile_complete(t *testing.T) {
+	if os.Getenv("ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID") == "" {
+		t.Skip("Skipping as ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID is not specified")
+		return
+	}
+	data := acceptance.BuildTestData(t, "azurerm_trusted_signing_certificate_profile", "test")
+	r := TrustedSigningCertificateProfileResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("certificates.0.created_date").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.enhanced_key_usage").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.expiry_date").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.serial_number").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.status").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.subject_name").Exists(),
+				check.That(data.ResourceName).Key("certificates.0.thumbprint").Exists(),
+				check.That(data.ResourceName).Key("status").Exists(),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func TestAccTrustedSigningCertificateProfile_update(t *testing.T) {
+	if os.Getenv("ARM_TEST_TRUSTED_SIGNING_IDENTITY_ID") == "" {
```

Comment:
Can we get a secondary identity id to test updating it?

Reply:
added force new to it...